build/BuildTargets.targets

```diff
@@ -1,6 +1,6 @@
 <Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <AvaloniaBuildTasksLocation>$(MSBuildThisFileDirectory)\..\src\Avalonia.Build.Tasks\bin\$(Configuration)\netstandard2.0\Avalonia.Build.Tasks.dll</AvaloniaBuildTasksLocation>
+    <AvaloniaBuildTasksLocation>$(MSBuildThisFileDirectory)\..\src\Avalonia.Build.Tasks\bin\$(Configuration)\$(TargetFramework)\Avalonia.Build.Tasks.dll</AvaloniaBuildTasksLocation>
```

Comment:
Now we're multitargeting `netstandard2.0;net6.0` hardcoding this to `netstandard2.0` doesn't work, as the version that gets built will depend on the target framework being compiled.

Reply:
Actually, no, reverted this change.